lib/ImportAttributeLabel.js

```diff
@@ -66,17 +66,16 @@ class ImportAttributeLabel extends ImportClass {
       tmpAttributeLabel[this.helper.namespace('Classification__c')] = node.classification
       tmpAttributeLabel[this.helper.namespace('Is_Localizable__c')] = node.localizable
       tmpAttributeLabel[this.helper.namespace('Is_Searchable__c')] = node.searchable
+      tmpAttributeLabel['Label__c'] = node.name
```

Comment:
No need for `namespace`?

Reply:
that is why we have reviews. 

<img src="https://media1.giphy.com/media/xULW8v7LtZrgcaGvC0/giphy.gif"/>

Reply:
updated